src/ucs/debug/memtrack.h

```diff
@@ -130,7 +129,16 @@ char *ucs_strndup(const char *src, size_t n, const char *name);
 #define ucs_malloc(_s, ...)                        malloc(_s)
 #define ucs_calloc(_n, _s, ...)                    calloc(_n, _s)
 #define ucs_realloc(_p, _s, ...)                   realloc(_p, _s)
-#define ucs_memalign(_b, _s, ...)                  memalign(_b, _s)
+#if HAVE_POSIX_MEMALIGN
```

Comment:
can yous pls remove memalign instances from UCX code instead of defining a wrapper?